lib/DraggableCore.es6

```diff
@@ -331,6 +345,10 @@ export default class DraggableCore extends React.Component {
     // We're on a touch device now, so change the event handlers
     dragEventFor = eventsFor.touch;
 
+    // Remove the event listener that stopped document scrolling
+    document.removeEventListener('touchmove', this.removeScroll, false);
```

Comment:
You mean to scroll a div inside the element itself? Because the touch handler is only on the draggable element.